src/devices/machine/wd2010.cpp

```diff
@@ -896,8 +912,13 @@ TIMER_CALLBACK_MEMBER(wd2010_device::deassert_read)
 		m_error &= ~ERROR_ID;
 		param &= ~STATUS_ERR;
 
-		m_out_bdrq_cb(0);
-		complete_immediate(param);
+		if ((m_task_file[TASK_FILE_COMMAND] & 4) && m_task_file[TASK_FILE_SECTOR_COUNT] > 1) {
+			m_task_file[TASK_FILE_SECTOR_COUNT]--;
+			m_task_file[TASK_FILE_SECTOR_NUMBER]++;
+			read_sector(m_task_file[TASK_FILE_COMMAND]);
+		} else {
+			complete_immediate(param);
+		}
```

Comment:
Please use Allman formatting like the existing code in this file.

Reply:
Ok.

Reply:
After finding all the various disk images required for DEC rainbow, i tried it, and it looks like DEC rainbow is no longer working with the wd2010 changes. So better drop this PR.